lib/repl.js

```diff
@@ -62,6 +62,7 @@ const {
   Boolean,
```

Comment:
To fix the lint error (this actually needs to be at the very top of the list to keep the list in ASCII order)
```suggestion
  ArrayPrototypeAt,
  Boolean,
```